README.md

```diff
@@ -71,9 +514,9 @@ public class StocksController
     }
 }
 ```
-### API documentation
+### Api Documentation
```

Comment:
Should we choose between "API documentation" and "ApiDocumentation" as the section title? See my overall review comment.

Reply:
Going off of what I wrote above, how do you feel about "Hydra Api Documentation"?

Reply:
Or maybe "Documentation of APIs and the Hydra ApiDocumentation"?